test/test_turbulence_closures.jl

```diff
@@ -193,8 +188,14 @@ end
 
     @testset "Closure instantiation" begin
         @info "  Testing closure instantiation..."
-        for closure in closures
-            @test closure_instantiation(closure)
+        for closurename in closures
+            closure = getproperty(TurbulenceClosures, closurename)()
+            @test closure isa TurbulenceClosures.AbstractTurbulenceClosure
+
+            grid = RectilinearGrid(CPU(), size=(1, 1, 1), extent=(1, 2, 3))
+            model = NonhydrostaticModel(grid=grid, closure=closure, tracers=:c)
+            @test diffusivity(model.closure, model.diffusivity_fields, Val(:c)) isa Union{Number, Field, AbstractOperations.AbstractOperation}
+            @test viscosity(model.closure, model.diffusivity_fields) isa Union{Number, Field}
```

Comment:
```suggestion
            c = model.tracers.c
            u = model.velocities.u
            κ = diffusivity(model.closure, model.diffusivity_fields, Val(:c)) 
            κ_dx_c = κ * ∂x(c)
            ν = viscosity(model.closure, model.diffusivity_fields)
            ν_dx_u = ν * ∂x(c)
            @test ν_dx_u[1, 1, 1] == 0.0
            @test κ_dx_c[1, 1, 1] == 0.0
```

A crucial problem with our tests is that they are hard to maintain, which slows down development. So I'm trying to help with that. First we have to make sure that when we add new tests, they are easy to maintain. We should also refactor as much as possible existing tests to make them easier to maintain.